vsc/baseline/inference.py

```diff
@@ -30,6 +30,8 @@ class InferenceTransforms(enum.Enum):
     RESIZE_288 = enum.auto()
     # Resize the short edge to 320, then take the center crop
     RESIZE_320_CENTER = enum.auto()
+    # Aspect-ratio preserving resize to 224x224
```

Comment:
My read is that this doesn't preserve the aspect ratio, and compresses the long side to form a square. That's a good preprocessing choice for DINO, but the comment shouldn't say "aspect-ratio preserving". We can just say "Resize to 224x224".

Reply:
Great catch. Fixed